Gemfile.lock

```diff
@@ -622,7 +622,7 @@ DEPENDENCIES
   webmock
 
 RUBY VERSION
-   ruby 3.2.2p53
+   ruby 2.7.8p225
```

Comment:
This isn't right is it?!

Reply:
Yeah, most of these auto generated PRs are broken.